docs/content/en/docs/references/cli/_index.md

```diff
@@ -539,6 +539,7 @@ Global Flags:
 ```
 Env vars:
 
+* `SKAFFOLD_XXENABLEJIBINIT` (same as `--XXenableJibInit`)
```

Comment:
Is there any way to hide this from the docs? Figured out how to do with with the CLI flag but not env vars.

Reply:
Strange, it should be hidden:  https://github.com/GoogleContainerTools/skaffold/blob/13a66b2d2ce927491844916bc0a80bbadf8564a8/cmd/skaffold/man/man.go#L49
Perhaps you have to rebase because it's a recent change